src/main/java/com/github/yusufyilmazfr/podcast4j/config/Config.java

```diff
@@ -21,12 +26,18 @@ public String getUserAgent() {
         return userAgent == null ? DEFAULT_USER_AGENT : userAgent;
     }
 
+    public ProxySelector getProxySelector() {
+        return proxySelector;
```

Comment:
When i run the tests, whole tests throw an exception for null value for `proxySelector`, i think we may define or not `proxySelector` in Config, this is an example of basic config:
```java
Config.builder()
                     .secret("secret")
                     .authKey("auth-key")
                     .userAgent("SuperPodcastPlayer/1.8")
                     .build();
```
maybe `getProxySelector` should return default `proxySelector ` if there is no any selector?
```java
    public ProxySelector getProxySelector() {
        return Optional.ofNullable(proxySelector).orElse(ProxySelector.getDefault());
    }
```

 What do you think?

Reply:
I just submited a commit, which resolve this issue more simple. Please have a look.